lib/internal/errors.js

```diff
@@ -272,6 +272,9 @@ E('ERR_BUFFER_TOO_LARGE',
   `Cannot create a Buffer larger than 0x${kMaxLength.toString(16)} bytes`);
 E('ERR_CANNOT_WATCH_SIGINT', 'Cannot watch for SIGINT signals');
 E('ERR_CHILD_CLOSED_BEFORE_REPLY', 'Child closed before reply received');
+E('ERR_CHILD_PROCESS_IPC_REQUIRED',
+  'Forked processes must have an IPC channel, %s is %s');
```

Comment:
This is used in one place, it doesn't feel like "%s is %s" is really that helpful. It should probably just say something like `Forked processes must have an IPC channel, missing value 'ipc' in %s`.